templates/ossec/config/rubber/deploy-ossec.rb

```diff
@@ -0,0 +1,70 @@
+namespace :rubber do
+
+  namespace :ossec do
+
+    rubber.allow_optional_tasks(self)
+
+    after "rubber:install_packages", "rubber:ossec:install"
+
+    task :install, :roles => :ossec do
+      rubber.sudo_script 'install_ossec', <<-ENDSCRIPT
+        if ! ps -e | grep ossec &> /dev/null; then
+          # Fetch the sources.
+          curl --header 'Host: www.ossec.net' --header 'User-Agent: Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:31.0) Gecko/20100101 Firefox/31.0' --header 'Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8' --header 'Accept-Language: en-US,en;q=0.5' --header 'DNT: 1' --header 'Referer: http://www.ossec.net/?page_id=19' --header 'Connection: keep-alive' 'http://www.ossec.net/files/ossec-hids-#{rubber_env.ossec_version}.tar.gz' -o 'ossec-hids-#{rubber_env.ossec_version}.tar.gz' -L
```

Comment:
You probably don't need all the header stuff, just

```wget -O 'ossec-hids-#{rubber_env.ossec_version}.tar.gz' 'http://www.ossec.net/files/ossec-hids-#{rubber_env.ossec_version}.tar.gz'``` (possibly with the follow links flag?)